<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog {
	background-color: #ffffff;
}

#QWidget{
	background: white;
	border-radius: 8px;
	box-shadow: 0 1px 3px rgba(0,0,0,0.1);
    border: 1px solid #e0e0e0;
}

#widget{
	background: white;
	border-radius: 8px;
    border: 1px solid #e0e0e0;
}

#widget_2{
	background: white;
	border-radius: 8px;
    border: 1px solid #e0e0e0;
}

#cctv_date, #weight_date {
	color: #007bff;
}

#weight_warning {
	color: #dc3545
}

#cctv_measure_btn, #weight_measure_btn {
	background: #007bff;
    color: white;
    border: none;
    border-radius: 4px;
}</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>자가진단 항목 선택</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>370</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{
	background: white;
	border-radius: 8px;
    border: 1px solid #e0e0e0;
}</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/Cam.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="cctv_measure">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>14</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CCTV 진단</string>
    </property>
   </widget>
   <widget class="QLabel" name="cctv_desc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>선택한 CCTV의 접속상태, 장치정보, 
통신 연결상태를 진단합니다.</string>
    </property>
   </widget>
   <widget class="QLabel" name="cctv_date">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_5 {
	color: #007bff;
}</string>
    </property>
    <property name="text">
     <string>최근 진단일시: ----.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cctv_measure_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton {
	background: #007bff;
    color: white;
    border: none;
    border-radius: 4px;
}</string>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cctv_radio1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>진입로</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cctv_radio2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>계근대</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cctv_radio3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>보편장</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>370</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_2{
	background: white;
	border-radius: 8px;
    border: 1px solid #e0e0e0;
}</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/Weight.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="weight_measure">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>14</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>계량값 진단</string>
    </property>
   </widget>
   <widget class="QLabel" name="weight_desc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>연결 설정값, 계량값 수신 상태,
계량값 튐현상을 진단합니다.</string>
    </property>
   </widget>
   <widget class="QLabel" name="weight_date">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_9 {
	color: #007bff;
}</string>
    </property>
    <property name="text">
     <string>최근 진단일시: ----.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weight_measure_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2 {
	background: #007bff;
    color: white;
    border: none;
    border-radius: 4px;
}</string>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
   </widget>
   <widget class="QLabel" name="weight_warning">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_10 {
	color: #dc3545
}</string>
    </property>
    <property name="text">
     <string>* 정확한 진단을 위해 계근대에
차량이 없을 때 측정하세요!</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="weight_radio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>계근대</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
